src/view/components/decisionScopes.jsx

```diff
@@ -183,21 +162,3 @@ const DecisionScopes = ({
     </div>
   );
 };
-
-DecisionScopes.propTypes = {
-  initInfo: PropTypes.object,
-  formikProps: PropTypes.object,
-  registerImperativeFormApi: PropTypes.func
-};
```

Comment:
This is interesting. Apparently DecisionScopes is no longer recognized as a React component, maybe because it's an anonymous function? I think I'd prefer keeping it as a named function for easier debugging when there's an error in the component, but what do you think? Also, do you feel like prop types are providing value for the cost of maintenance?

Reply:
I think prop types are valuable. It has led to better error messages and easier debugging for me on a number of occasions.